premerge error with flake8

new issue from flake8-bugbear: 23.1.14
```
6.0.0 (flake8-bugbear: 23.1.14, flake8-comprehensions: 3.10.1,
flake8-executable: 2.1.2, mccabe: 0.7.0, pep8-naming: 0.13.3, pycodestyle:
2.10.0, pyflakes: 3.0.1) CPython 3.8.15 on Linux
/home/runner/work/MONAI/MONAI/monai/apps/auto3dseg/bundle_gen.py:164:29: B028 'config_yaml' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/mmars/mmars.py:229:17: B028 'item.get()' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/mmars/mmars.py:273:17: B028 'model_dir' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/pathology/handlers/prob_map_producer.py:116:26: B028 'name' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/pathology/transforms/post/array.py:135:21: B028 'activation' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/pathology/transforms/post/array.py:140:30: B028 'type()' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/pathology/transforms/post/array.py:783:21: B028 'activation' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/pathology/transforms/post/array.py:788:30: B028 'type()' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/pathology/transforms/post/dictionary.py:207:28: B028 'self.border_key' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/pathology/transforms/post/dictionary.py:243:28: B028 'self.dist_map_key' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/pathology/transforms/post/dictionary.py:292:28: B028 'self.markers_key' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/utils.py:147:17: B028 '_basename()' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/apps/utils.py:188:21: B028 'filepath' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/config_item.py:333:27: B028 'import_string' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/config_parser.py:397:30: B028 'filepath' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/reference_resolver.py:127:28: B028 'id' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/reference_resolver.py:144:34: B028 'd' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/reference_resolver.py:144:34: B028 'id' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/reference_resolver.py:231:27: B028 'ref_id' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/reference_resolver.py:[240](https://github.com/Project-MONAI/MONAI/actions/runs/3919924032/jobs/6701234686#step:7:241):49: B028 'ref_id' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/scripts.py:987:30: B028 'filename' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/scripts.py:1039:26: B028 'str()' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/scripts.py:1042:26: B028 'str()' is manually surrounded by quotes, consider using the `!r` conversion flag.
/home/runner/work/MONAI/MONAI/monai/bundle/utils.py:183:26: B028 'bundle_path' is manually surrounded by quotes, consid
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -143,7 +143,7 @@ max_line_length = 120
 # E501 is not flexible enough, we're using B950 instead
 # N812 lowercase 'torch.nn.functional' imported as non lowercase 'F'
 # B023 https://github.com/Project-MONAI/MONAI/issues/4627
-# B208 https://github.com/Project-MONAI/MONAI/issues/5855
+# B028 https://github.com/Project-MONAI/MONAI/issues/5855
 ignore =
     E203
     E501
@@ -154,7 +154,7 @@ ignore =
     N812
     B023
     B905
-    B208
+    B028
 per_file_ignores = __init__.py: F401, __main__.py: F401
 exclude = *.pyi,.git,.eggs,monai/_version.py,versioneer.py,venv,.venv,_version.py
 

--- a/setup.cfg
+++ b/setup.cfg
@@ -143,6 +143,7 @@ max_line_length = 120
 # E501 is not flexible enough, we're using B950 instead
 # N812 lowercase 'torch.nn.functional' imported as non lowercase 'F'
 # B023 https://github.com/Project-MONAI/MONAI/issues/4627
+# B208 https://github.com/Project-MONAI/MONAI/issues/5855
 ignore =
     E203
     E501
@@ -153,6 +154,7 @@ ignore =
     N812
     B023
     B905
+    B208
 per_file_ignores = __init__.py: F401, __main__.py: F401
 exclude = *.pyi,.git,.eggs,monai/_version.py,versioneer.py,venv,.venv,_version.py
 

